### PR TITLE
Header iterations

### PR DESCRIPTION
**Pull request by RileyManda**
I have completed the task of updating the header component:🌱 

**The following implementations were made:** 🌱 

- [x] Re-aligned logo to match the [UI sample provided](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md)
- [x] Updated font size and alignment of app-text
- [x] Implemented bold font weight to app-text
- [x] Implemented horizontal line

In addition, I have ensured that this project fulfills the following default requirements :
✅  - The application behaves the same way after refactoring
✅ - There are [no linter errors](https://github.com/microverseinc/linters-config).
✅ - Used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
✅  - Documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
✅  - Followed the list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and  [JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

Please review my implementation and suggest any improvements and changes.

Thank you 🙏

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -6,15 +6,23 @@
...
 
+/* Nav header  */
 .logo-container {
   display: flex;
   align-items: center;
+  margin-top: 10px;
+  margin-left: 17px;
 }
 
 .app-logo {
```
Comment (yaredtekle022) [inline src/App.css:17]: You updated your CSS file before pulling from dev branch. you dev branch is updated with rockets-list branch from your coding partners side

File: src/App.js [hunk 2]
```diff
@@ -13,6 +14,7 @@ function App() {
   return (
     <BrowserRouter>
       <Header />
+      <HorizontalLine />
```
Comment (yaredtekle022) [inline src/App.js:17]: Please pull from dev before pushing your changes

**Review by yaredtekle022 — APPROVED** [review]
Hi [@RileyManda](url) ou did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

Top Highlight 🎉

- PR has a good title and summary.
- Your Design looks awesome, keep rocking 🚀


### STATUS: CHANGES REQUESTED ♻️

Please Check the comments under the review and make all required changes

Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers 🥂 and Happy coding!!! 👯💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.css [hunk 3]
```diff
@@ -6,15 +6,23 @@
   align-items: center;
 }
 
+/* Nav header  */
 .logo-container {
   display: flex;
   align-items: center;
+  margin-top: 10px;
+  margin-left: 17px;
 }
```
Comment (yaredtekle022) [inline src/App.css:15]: Please pull before pushing your branch 👍

**Review by yaredtekle022 — CHANGES_REQUESTED** [review]
Hi [@RileyManda](https://github.com/RileyManda/space-travelers-hub/pull/url) ou did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

Top Highlight 🎉

    PR has a good title and summary.
    Your Design looks awesome, keep rocking 🚀

STATUS: CHANGES REQUESTED ♻️

Please Check the comments under the review and make all required changes

Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers 🥂 and Happy coding!!! 👯💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.css (hunk 3 above)
Comment (RileyManda) [reply src/App.css:15]: Hi @yaredtekle022 
This is an independent header component. I created the branch from dev following the correct gitflow. The only changes in this iteration are the horizontal line and logo and text alignment of the header.No other changes have been implemented.

**Review by yaredtekle022 — APPROVED** [review]
Hi @RileyManda 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Highlights

- logo aligned per design
- Descriptive pull request message


Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form

**Pull request merged**